prow/cmd/checkconfig/main.go

```diff
@@ -1059,3 +1065,35 @@ func validateTideContextPolicy(cfg *config.Config) error {
 
 	return utilerrors.NewAggregate(errs)
 }
+
+func validateClusterFieldForJobBase(job config.JobBase) error {
```

Comment:
`ForJobBase` stutters (in both signature and callsite). How about calling this one `validateJobCluster`?

Reply:
Updated.